rfc6962-bis.xml

```diff
@@ -351,18 +351,30 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
       <section title="Redaction of Domain Name Labels" anchor="redacting_labels">
         <section title="Redacting Labels in Precertificates">
           <t>
-            When creating a precertificate, the CA MAY substitute one or more labels in each DNS-ID and <xref target="RFC6125">CN-ID</xref> with a corresponding number of <spanx style="verb">?</spanx> labels. Every label to the left of a <spanx style="verb">?</spanx> label MUST also be redacted. For example, if a certificate contains a DNS-ID of <spanx style="verb">top.secret.example.com</spanx>, then the corresponding DNS-ID in the precertificate could contain <spanx style="verb">?.?.example.com</spanx> instead, but not <spanx style="verb">top.?.example.com</spanx> instead.
+            When creating a precertificate, the CA MAY omit the <xref target="RFC5280">subjectAltName</xref> extension. Instead, the CA MUST include a <xref target="redacted_san_extension">Redacted SAN</xref> extension, containing the entries that will be included in the certificate's subjectAltName extension, in a redacted form.
           </t>
           <t>
-            Wildcard <spanx style="verb">*</spanx> labels MUST NOT be redacted. However, if the complete leftmost label of a DNS-ID or CN-ID is <spanx style="verb">*</spanx>, it is considered redacted for the purposes of determining if the label to the right may be redacted. For example, if a certificate contains a DNS-ID of <spanx style="verb">*.top.secret.example.com</spanx>, then the corresponding DNS-ID in the precertificate could contain <spanx style="verb">*.?.?.example.com</spanx> instead, but not <spanx style="verb">?.?.?.example.com</spanx> instead.
+            One or more labels in each <xref target="RFC6125">DNS-ID</xref> can be replaced with <spanx style="verb">REDACTED(label)</spanx> labels.
+          </t>
+          <t>
+            The redacted label is defined as follows:
+            <vspace blankLines='1' />
+            REDACTED(label) = ? || HASH_INDEX || ? || HEX(HASH(serial_number || label))
```

Comment:
@robstradling I like encoding hash_index as a hex-encoded byte.  It will simplify the client by avoiding the need to decode an integer between the ? marks.